types/set/README.md

```diff
@@ -0,0 +1,137 @@
+# Set
+
+_Module_
+
+Implements a very simple `SetInt` struct.
+
+<br>
+This is NOT a Mojo builtin, so it needs to be imported
+<br>
+
+### **Parameters:**
+
+-   None
+
+### **Fields:**
+
+-   None
+
+### **Functions:**
+
+#### `__init__`
+
+> `fn __init__(inout self) -> None:`
+
+Constructs a new `SetInt` with a default capacity (capacity_default = 11).
+
+##### **Args:**
+
+-   `None`
+
+##### **Returns:**
+
+-   `None`
+
+#### `__init__`
+
+> `fn __init__(inout self) -> None:`
+
+Constructs a new `SetInt` with a default capacity (capacity_default = 11).
+
+##### **Args:**
+
+-   `None`
+
+##### **Returns:**
+
+-   `None`
+
+#### `__del__`
+
+> `fn __del__(owned self) -> None:`
+
+Destructs a `SetInt`.
+
+##### **Args:**
+
+-   `None`
+
+##### **Returns:**
+
+-   `None`
+
+#### `contains`
+
+> `fn contains(self, value: Int) -> Bool`
+
+Checks if value is already in `SetInt`.
+
+##### **Args:**
+
+-   value(`Int`): The value to check.
+
+##### **Returns:**
+
+-   `Bool`: True if value is in `SetInt`, False otherwise.
+
+#### `add`
+
+> `fn add(self, value: Int) -> None:`
+
+Add a value to `SetInt`.
+
+##### **Args:**
+
+-   value(`Int`): The value to add.
+
+##### **Returns:**
+
+-   `None`
+
+#### `remove`
```

Comment:
What happens if I have two of the same Int in the set and try to remove it? Will it remove all instances?

Reply:
The code shows it deletes all instances (I think) but just adding that to the readme would be nice.

Reply:
there can only exist one instance of a value in a set, so it only removes one